makefiles/mcuboot.mk

```diff
@@ -12,6 +12,8 @@ MCUBOOT_BIN_MD5 ?= 0c71a0589bd3709fc2d90f07a0035ce7
 
 export IMAGE_HDR_SIZE ?= 512
 
+BUILDDEPS += $(MCUBOOT_KEYFILE) $(MCUBOOT_BIN) mcuboot
```

Comment:
However, we discussed it IRL, none of these files should be declared as `BUILDDEPS` they are not build dependencies, they just need to be done after `clean` if it is there.

The reason being that they are in the `BINDIR` directory that can be deleted.

I will propose a pre-first fix to go in the direction of adding this `BINDIR` variable and target.